src/models/schemas.js

```diff
@@ -0,0 +1,52 @@
+const account = {
```

Comment:
Can you remove this file for now? I don't think this is how we want to do this.

Reply:
okay